src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,4 +1,15 @@
-// Add the required types and props
-export const UserInfo = () => (
-  <>UserInfo markup</>
-);
+import { User } from '../../types/User';
+
+type Props = {
+  user: User
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const { email, name } = user;
+
+  return (
+    <a className="UserInfo" href={`mailto:${email}`}>
```

Comment:
```suggestion
    <a 
      className="UserInfo" 
      href={`mailto:${email}`}
    >
```
it's better to do it like this if you want to